testsuite/integration-arquillian/servers/auth-server/quarkus/src/main/content/conf/keycloak.properties

```diff
@@ -34,4 +34,7 @@ spi.theme.folder.dir=${kc.home.dir:}/themes
 spi.password-policy.password-blacklist.blacklists-path=${kc.home.dir:}/dependency/password-blacklists
 
 # http client connection reuse settings
-spi.connections-http-client.default.reuse-connections=false
\ No newline at end of file
+spi.connections-http-client.default.reuse-connections=false
+
+# set max-length of event representation stored so the db can handle it
+spi.events-store.jpa.max-detail-length=2000
```

Comment:
@pedroigor i could add that to the dists keycloak.properties, too, if needed. seems reasonable for me to have it set by default to a value which does not cause the db to throw an exception.

Reply:
If we any default value set in the Wildfly dist, I agree. Otherwise, I'm not sure.

Reply:
Looked it up, we have not. The lets leave it like this for now.